app/src/main/java/com/vitorpamplona/amethyst/ui/qrcode/QrCodeDrawer.kt

```diff
@@ -34,13 +33,13 @@ fun QrCodeDrawer(contents: String, modifier: Modifier = Modifier) {
         createQrCode(contents = contents)
     }
 
-    val foregroundColor = MaterialTheme.colors.onSurface
+    val foregroundColor = Color.Black
 
     Box(
         modifier = modifier
             .defaultMinSize(48.dp, 48.dp)
             .aspectRatio(1f)
-            .background(MaterialTheme.colors.background)
+            .background(Color.White)
```

Comment:
QR code spec says that the codes must be dark on a light background. With the previous version, our QR codes were improperly formatted in dark mode.

Also note that zxing does not support QR codes that aren't up to spec.

Reply:
Are you sure this isn't an old spec? The last time I read the text, it could be high-contrast color.

Reply:
> old spec

That could be.

Reply:
That can be a problem. Damus uses their blue-purple GRADIENT as the QR code background :(

Reply:
Do you want to try to separate this work into a full-on flavor of the app? Even if it doesn't work that well, F-Droid folks would love it. 

The idea is: the current package name has Google dependencies and is shipped to the PlayStore. The new flavor would remove Google's deps and add this work and ship to F-Droid

Reply:
Like here: https://stackoverflow.com/questions/18196974/how-to-define-different-dependencies-for-different-product-flavors